pkgs/tools/networking/trurl/default.nix

```diff
@@ -1,27 +1,40 @@
-{ lib, stdenv, fetchFromGitHub, curl, perl }:
+{ lib, stdenv, fetchFromGitHub, installShellFiles, curl, python3 }:
 
 stdenv.mkDerivation rec {
   pname = "trurl";
-  version = "0.4";
+  version = "0.5";
 
   src = fetchFromGitHub {
     owner = "curl";
     repo = pname;
     rev = "${pname}-${version}";
-    sha256 = "sha256-f9z0gPtHcn3iWFA6MT6ngok0nXBcd6zJ8Tjnb5Lzf6c=";
+    sha256 = "sha256-ho/XEuLQoUPTQNUZpen7WdJf0Iz0RXHrh+VbbrS8c9c=";
   };
 
   separateDebugInfo = stdenv.isLinux;
 
   enableParallelBuilding = true;
 
+  preBuild = ''
+    # Fix up version number
+    substituteInPlace version.h --replace 0.4 0.5
+  '';
+
   buildInputs = [ curl ];
```

Comment:
```suggestion
  nativeBuildInputs = [ curl ];

  buildInputs = [ curl ];

```

Reply:
There is a reason to add `curl` to `nativeBuildInputs` as well? 
Maybe I misunderstood something about `{native,}BuildInputs`, but the trurl build process doesn't need host curl, only for the target.

Reply:
https://github.com/curl/trurl/commit/a7f9ca9c42c2cf29cc84becb38b56082898868d4 switches to using `curl-config`, which should be in `nativeBuildInputs`, you can check this with `strictDeps = true`

Reply:
Oh right, I overlooked, sorry!